src/explore-education-statistics-admin/src/modules/find-statistics/components/__tests__/__snapshots__/EditableContentSubClockRenderer.test.tsx.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`EditableContentSubBlockRenderer Renders editable Markdown block correctly 1`] = `
+
+<div class="editable-content  ">
+  <div class="editable-button">
+  </div>
+  <div>
+    <p classname="govuk-body">
+    </p>
+    <p>
+      test
+    </p>
+    <p>
```

Comment:
Are we supposed to have an extra set of `p`s here?

Reply:
yes, it's content from the API. It's generated from the markup.

Reply:
Wait so I'm a bit confused, we've also got a random `<p classname="govuk-body">` as well just above it. I can appreciate the 'test' content is coming from the API, but these extraneous `p`s seem incorrect

Reply:
I'm intrigued by it all. If I changed it to a div, this is spat out...

```
<div classname="govuk-body">
      <p>
        test
      </p>
    </div>
```

Reply:
And this is what the HTML is trying to be set to...
```
               <p className="govuk-body"><p>test</p>
       </p>
```
by the code

Reply:
ah... nested <p> tags